### PR TITLE
NTP-879: School user is informed when a response is received

### PR DESCRIPTION
**Pull request by hasan3d**
## Things to check

- [ ] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [ ] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [ ] `dotnet format` has been run in the repository root
- [ ] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [ ] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: Application/Commands/SendEnquirerEnquiryResponseReceivedEmailCommand.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    {
+        var enquirerEnquiryResponseReceivedData =
+            await _unitOfWork.MagicLinkRepository.
+                GetEnquirerEnquiryResponseReceivedData(request.Data.EnquiryId!, request.Data.TuitionPartnerId);
+
+        if (enquirerEnquiryResponseReceivedData == null)
+        {
+            _logger.LogError("Unable to send enquirer enquiry response received email. " +
+                             "Can't find magic link token with the type {type} by enquiry Id {enquiryId}",
+                request.Data.EnquiryId, MagicLinkType.EnquirerViewAllResponses.ToString());
```
Comment (kieranohanlon) [inline Application/Commands/SendEnquirerEnquiryResponseReceivedEmailCommand.cs:46]: Are these vars the wrong way around?

File: vars-qa.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ google-analytics-measurement-id:
 dfe-analytics-dataset: fatp_events_qa
 govuk-notify-enquiry-submitted-confirmation-to-enquirer-template-id: b1e29214-dfc0-4225-b65a-df6a20c4c297
 govuk-notify-enquiry-submitted-to-tp-template-id: b24ffca7-6119-4dbf-8ae2-dceb2c300ca4
+govuk-notify-enquiry-response-received-confirmation-to-enquirer-template-id: f64db52d-08ce-4f2a-9b37-a374db1ad77a
```
Comment (kieranohanlon) [inline vars-qa.yml:12]: When adding in the other templates IDs I've also added them to the other var yml files (staging, research, production etc).  Is it worth adding this one in to all vars too?  If not then you'll just need to remember to do it at some point in the future.

File: Application/Commands/SendEnquirerEnquiryResponseReceivedEmailCommand.cs (hunk 1 above)
Comment (hasan3d) [reply Application/Commands/SendEnquirerEnquiryResponseReceivedEmailCommand.cs]: resolved!

File: vars-qa.yml (hunk 2 above)
Comment (hasan3d) [reply vars-qa.yml:12]: Cool - I have added that for the rest of the `yml` files (staging, research, production).

**Pull request merged**